src/components/game/GameContainer.js

```diff
@@ -59,8 +78,8 @@ class GameBattle extends Component {
       // </div>)
       <div>
       <div className="Boards">
-        <Board />
-        <GuessBoard />
+        <Board board={this.props.boards.board}/>
+        <GuessBoard player={player} board={this.props.boards.guessBoard}/>
```

Comment:
ahhh so that's how the magic is done!! yaaaay

Reply:
Oops! I still have put board for the guessBoard as well :stuck_out_tongue:. Yeah! I just wanted to just pass them as prop from the parent component